.github/workflows/python.yaml

```diff
@@ -1,28 +1,55 @@
 name: Python package
 
-on: [push]
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      #Trigger workflow on push to any branch or branch heirarchy:
```

Comment:
...This is why I stick to math. 😄 

Also, it looks like this is a copy/paste typo because it's spelt the same in the capgen unit tests workflow yaml.  Should I update that one as well?

Reply:
Updated!

Reply:
thank you!